### PR TITLE
Adds RegisterServlet and Creates Register.jsp.

### PR DESCRIPTION
**Pull request by aditija**

### CODE REVIEW

**Comment by erschwartz** [comment]
lgtm

**Comment by pavulurisai** [comment]
LGTM

File: src/main/java/codeu/controller/RegisterServlet.java [hunk 1]
```diff
@@ -0,0 +1,19 @@
...
+ */
+public class RegisterServlet extends HttpServlet {
+
+ @Override
+     public void doGet(HttpServletRequest request, HttpServletResponse response)
+     throws IOException, ServletException {
+
+     request.getRequestDispatcher("/WEB-INF/view/register.jsp").forward(request, response);
+ }
+}
```
Comment (pavulurisai) [inline src/main/java/codeu/controller/RegisterServlet.java]: Remove the character after the closing } on line # 19

Comment (aditija) [reply src/main/java/codeu/controller/RegisterServlet.java]: Done!

**Pull request merged**